src/components/FoodList.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react'
+import FoodBox from './FoodBox';
+
+export default class FoodList extends Component {
+    render() {
+        const {foods} = this.props;
+
+        return (
+            <div>
+                {
+                    foods.map((item, i) => {
```

Comment:
making this a component may not be needed. since right now it is only running the map. Instead you could run the map in App.js and avoid this part. The result will be the same with way less code.